ros_buildfarm/workspace.py

```diff
@@ -99,6 +100,13 @@ def call_build_tool(
     if cmake_args:
         cmd += ['--cmake-args'] + cmake_args
 
+    if ctest_args:
+        if build_tool == 'colcon':
+            cmd += ['--ctest-args'] + ctest_args
+        # No ctest parameters are passed in catkin_make_isolated
+        # because in ROS 1 the test target of the makefile is being
+        # not ctest
```

Comment:
Looks like you might have accidentally a whole word?

```suggestion
        # invoked not ctest
```